watson/watson.py

```diff
@@ -425,3 +433,11 @@ def merge_report(self, frames_with_conflict):
                 merging.append(conflict_frame)
 
         return conflicting, merging
+
+    def _default_tags(self, project):
+        default_tags_raw = self.config.get('default_tags', project)
+        if default_tags_raw is None:
+            default_tags = []
+        else:
+            default_tags = shlex.split(default_tags_raw)
```

Comment:
I would prefer something like:

```python
default_tags = []
if default_tags_raw is not None:
    default_tags = shlex.split(default_tags_raw)
```

I'm trying to respect Object Calisthenics, see [rule 2](http://williamdurand.fr/2013/06/03/object-calisthenics/#2-don-t-use-the-else-keyword). Poke @willdurand :stuck_out_tongue_winking_eye: